vnext/Microsoft.ReactNative/Fabric/ActivityIndicatorComponentView.cpp

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
nit: looks like accidental whiteline add

Reply:
this one is a removal :) we had a whiteline there for no reason